### PR TITLE
send new api file link to end users

### PR DESCRIPTION
**Pull request by sastels**
# Summary | Résumé

https://app.zenhub.com/workspaces/notify-planning-614b3ad91bc2030015ed22f5/issues/cds-snc/notification-planning/240

This PR sends sets the end user's file link to the new api file link (#65)

**Note**: requires https://github.com/cds-snc/notification-manifests/pull/1182

# Test instructions | Instructions pour tester la modification

- Run api and document-download-api locally
- send yourself files, both attached and as links

### CODE REVIEW

File: app/download/views.py [hunk 1]
```diff
@@ -51,7 +51,7 @@ def download_document_b64(service_id, document_id):
         abort(404)
 
     filename = request.args.get('filename')
-    sending_method = request.args.get('sending_method')
+    sending_method = request.args.get('sending_method', 'link')
```
Comment (sastels) [inline app/download/views.py:54]: `None` is treated as `link`. Making it explicit here allows the url returned by `url_for(
        'download.download_document_b64',...)` to be simpler.

File: app/download/views.py [hunk 2]
```diff
@@ -13,7 +13,7 @@ def download_document(service_id, document_id):
         return jsonify(error='Missing decryption key'), 400
 
     filename = request.args.get('filename')
-    sending_method = request.args.get('sending_method')
+    sending_method = request.args.get('sending_method', "link")
```
Comment (sastels) [inline app/download/views.py:16]: `None` is treated as "link" later on. Making it explicit here allows the url returned by url_for( 'download.download_document_b64',...) to be simpler.

File: tests/utils/test_urls.py [hunk 3]
```diff
@@ -1,29 +1,29 @@
...
+    ) == 'http://localhost:7000/d/{}/{}?key={}'.format(
         'AAAAAAAAAAAAAAAAAAAAAA',
         'AAAAAAAAAAAAAAAAAAAAAQ',
         SAMPLE_B64
     )
 
 
-def test_get_frontend_download_url_returns_frontend_url_with_filename(app):
-    assert get_frontend_download_url(
+def test_get_api_download_url_returns_frontend_url_with_filename(app):
```
Comment (jimleroyer) [inline tests/utils/test_urls.py:22]: Maybe we should rename the test as well as in contains a reference to a front-end URL.

**Review by jimleroyer — APPROVED** [review]
Looks excellent overall. Left a comment on potentially renaming a test.

File: tests/utils/test_urls.py (hunk 3 above)
Comment (sastels) [reply tests/utils/test_urls.py]: Good catch!

**Pull request merged**